mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AnnotationMapperTest.java

```diff
@@ -68,6 +69,8 @@ public final void setProducer(IProducer<T> producer) throws MailboxException {
         this.producer = producer;
         this.annotationMapper = newInstance.createAnnotationMapper();
         this.mailboxId = newInstance.generateId();
+
+        Assume.assumeNotNull(annotationMapper);
```

Comment:
I don't like assuming on implicit things, can't we assume on capabilities, I don't understand this modification